spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaClientAutoConfiguration.java

```diff
@@ -28,14 +28,17 @@
 import org.springframework.web.reactive.function.client.WebClient;
 import org.springframework.web.reactive.function.client.WebClient.Builder;
 
+import com.linecorp.armeria.spring.AbstractArmeriaClientAutoConfiguration;
+import com.linecorp.armeria.spring.ArmeriaClientConfigurator;
+
 /**
  * An auto-configuration for Armeria-based {@link WebClient}.
  */
 @Configuration
 @ConditionalOnClass(WebClient.Builder.class)
 @ConditionalOnMissingBean(ClientHttpConnector.class)
 @Import({ WebClientAutoConfiguration.class, DataBufferFactoryWrapperConfiguration.class })
-public class ArmeriaClientAutoConfiguration {
+public class ArmeriaClientAutoConfiguration extends AbstractArmeriaClientAutoConfiguration {
```

Comment:
Maybe it's time to introduce a new artifact `boot2-autoconfigure-common`?
So that we don't copy the source code using the script when building boo2-webflux-autoconfigure?
We still have to copy the source code for boot1 though.
/cc @trustin, @ikhoon, @anuraaga

Reply:
I also agree. 👍

Reply:
There are so many things to consider not to bring the breaking changes. 😅 
I might work on this before we release 1.4.0 so I think we can just focus on finishing this PR. 😄